libbpf-tools/offcputime.c

```diff
@@ -281,6 +290,30 @@ static void print_map(struct ksyms *ksyms, struct syms_cache *syms_cache,
 	free(ip);
 }
 
+static void print_headers()
+{
+	int i;
+
+	printf("Tracing off-CPU time (us) of");
+
+	if (env.pids[0]) {
+		printf(" PID [");
+		for (i = 0; i < MAX_PID_NR && env.pids[i]; i++)
+			printf("%d%s", env.pids[i], (i < MAX_PID_NR - 1 && env.pids[i + 1]) ? ", " : "]");
+	} else if (env.tids[0]) {
+		printf(" TID [");
+		for (i = 0; i < MAX_TID_NR && env.tids[i]; i++)
+			printf("%d%s", env.tids[i], (i < MAX_TID_NR - 1 && env.tids[i + 1]) ? ", " : "]");
+	} else {
```

Comment:
Should we print both PID and TID in the above?

Reply:
it's modified. thank you.